biogeochem/EDPhysiologyMod.F90

```diff
@@ -1332,33 +1332,34 @@ subroutine CWD_Input( currentSite, currentPatch)
          currentPatch%cwd_BG_in(c) = currentPatch%cwd_BG_in(c) + (struct_c + sapw_c) * & 
               SF_val_CWD_frac(c) * dead_n * (1.0_r8-EDPftvarcon_inst%allom_agb_frac(currentCohort%pft))
          
-         ! Send AGB component of boles from non direct-logging activities to AGB litter pool
+         ! Send AGB component of boles from both direct and non-direct logging activities to AGB litter pool
          if (c==ncwd) then
             
             ! CWD contributed by indirect damage
             currentPatch%cwd_AG_in(c) = currentPatch%cwd_AG_in(c) + (struct_c + sapw_c) * & 
                  SF_val_CWD_frac(c) * (dead_n_natural+ dead_n_ilogging) * &
                  EDPftvarcon_inst%allom_agb_frac(currentCohort%pft)
-            
+
             ! CWD contributed by logged boles due to losses in transportation
             currentPatch%cwd_AG_in(c) = currentPatch%cwd_AG_in(c) + &
```

Comment:
yay, I love it when people shorten lines

Reply:
:-)